### ISSUE TITLE
summary2 fails with MNLogit

### ISSUE THREAD

**Issue by josef-pkt** [statsmodels/statsmodels#966]
after running "\statsmodels\examples\example_discrete.py"

found doing some checking in PR #965

```
>>> print mlogit_res.summary2()
Traceback (most recent call last):
  File "<pyshell#4>", line 1, in <module>
    print mlogit_res.summary2()
  File "e:\josef\eclipsegworkspace\statsmodels-git\statsmodels-all-new2\statsmodels\statsmodels\discrete\discrete_model.py", line 2687, in summary2
    confint[i])
  File "e:\josef\eclipsegworkspace\statsmodels-git\statsmodels-all-new2\statsmodels\statsmodels\iolib\summary2.py", line 332, in summary_params
    data = np.hstack([data, conf_int])
  File "C:\Python26\lib\site-packages\numpy\core\shape_base.py", line 258, in hstack
    return _nx.concatenate(map(atleast_1d,tup),1)
ValueError: array dimensions must agree except for d_0
```

**Comment by amillner** [comment]
Any chance this on the list of things to be fixed? I'm looking to pull out confidence intervals and it fails and it looks like from the error message you posted the confidence intervals are the issue.

**Comment by josef-pkt** [comment]
I added labels so it doesn't get lost again in our large list of issues.
This was not my area, but I will look into it.

(MNLogit and VAR are outliers in the class hierarchy because they have 2-dimensional params, so need some special casing.)

**Comment by josef-pkt** [comment]
fixed in #3661 based on newer issues
